DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,7 +16,10 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="33">
+  <class name="pat::Electron"  ClassVersion="36">
+   <version ClassVersion="36" checksum="3720919820"/>
+   <version ClassVersion="35" checksum="905410242"/>
+   <version ClassVersion="34" checksum="1857282274"/>
```

Comment:
you should keep the checksum of what was 36 before and use it on v 34.
My instruction had a typo, it should have said ``` drop 35 and 34; and then rename 36 to 34 ``` (it should be followed in order).